### PR TITLE
docs: Add PRs to my profile page

### PR DESCRIPTION
**Pull request by YipZong**
Add PRs to my profile page, as instructed from jiarui

### CODE REVIEW

File: jekyll/_contributors/yipzong.md [hunk 1]
```diff
@@ -9,3 +9,9 @@ email: yip.zong@gmail.com
 ## Contact me
 
 - Email：<yip.zong@gmail.com>
+
+## My Contributions
+
+- <https://github.com/wechaty/wechaty.js.org/pull/1143>
```
Comment (lijiarui) [inline jekyll/_contributors/yipzong.md]: It would be better to add a title for PR.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_contributors/yipzong.md (hunk 1 above)
Comment (YipZong) [reply jekyll/_contributors/yipzong.md]: Added. Please see the latest commit.

File: jekyll/_contributors/yipzong.md [hunk 2]
```diff
@@ -9,3 +9,9 @@ email: yip.zong@gmail.com
 ## Contact me
 
 - Email：<yip.zong@gmail.com>
+
+## My Contributions
+
+- Add doc for add-contact-into-room, change roomTopic <https://github.com/wechaty/wechaty.js.org/pull/1143>
```
Comment (lijiarui) [inline jekyll/_contributors/yipzong.md]: I think the following would be better:

```
[Add doc for add-contact-into-room, change roomTopic](https://github.com/wechaty/wechaty.js.org/pull/1143)
```

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_contributors/yipzong.md (hunk 2 above)
Comment (YipZong) [reply jekyll/_contributors/yipzong.md]: OK! Fixed the style

**Review by lijiarui — APPROVED** [review]
LGTM

File: jekyll/_contributors/yipzong.md [hunk 3]
```diff
@@ -9,3 +9,9 @@ email: yip.zong@gmail.com
 ## Contact me
 
 - Email：<yip.zong@gmail.com>
+
+## My Contributions
+
+- [Add doc for add-contact-into-room, change roomTopic] (https://github.com/wechaty/wechaty.js.org/pull/1143)
```
Comment (lijiarui) [inline jekyll/_contributors/yipzong.md]: Still have a problem, you should remove the useless blank here: `] (`

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
Please make GitHub action green, when it turn green, I will keep review this.

File: jekyll/_contributors/yipzong.md (hunk 3 above)
Comment (YipZong) [reply jekyll/_contributors/yipzong.md]: OK! removed

**Comment by YipZong** [comment]
Green now :)

**Comment by lijiarui** [comment]
Good job！ Cheers!

**Pull request merged**